Project.toml

```diff
@@ -66,6 +68,7 @@ EinExprs = "0.5, 0.6"
 FiniteDifferences = "0.12"
 GraphMakie = "0.4,0.5"
 Graphs = "1.7"
+ITensorMPS = "0.2.6"
```

Comment:
mmm if we set the patch version, will we have problems when v0.2.7 lands?

like, will it force it to use v0.2.6 or will it be compatible (but not included) up to v0.3.0?

in principle we should only mark major and minor versions, not patch version (unless a patch is completely required for it to work)

Reply:
I think we should try to just set the major.minor version. If you need a higher patch version, you should just call `Pkg.update()` then.